packages/atomic/cypress/utils/componentUtils.ts

```diff
@@ -9,8 +9,12 @@ export function doSortAlphanumeric(originalValues: string[]) {
 }
 
 export function doSortOccurrences(originalValues: string[]) {
-  const getNumericalValue = (value: string) =>
-    parseInt(value.replaceAll(/[,()]/g, ''));
+  const getNumericalValue = (value: string) => {
+    const isKilo = value.includes('K') || value.includes('k');
+    const numericalVal = parseFloat(value.replaceAll(/[,(),Kk]/g, ''));
+    return isKilo ? numericalVal * 1000 : numericalVal;
+  };
```

Comment:
🤔 what is that?

Reply:
Previous getNumericalValue outputted wrong results. For a value such as 72.5K it would output 72 or a nan value instead of 72500.